Kiosk/Storyboards/StoryboardIdentifiers.swift

```diff
@@ -20,6 +20,11 @@ public enum SegueIdentifier : String {
     case ShowAdminOptions = "Show Admin Options"
     case ShowLoadingView = "Show Loading View"
     case ShowSaleArtworkDetails = "Show Sale Artwork Details"
+    case UNUSEDManualcreditcard = "UNUSED Manual credit card"
+    case UNUSEDPushtoSwipeCreditCard = "UNUSED Push to Swipe Credit Card"
+    case UNUSEDPushtoemail = "UNUSED Push to email"
+    case UNUSEDPushtoentermobilenumber = "UNUSED Push to enter mobile number"
+    case UNUSEDPushtopassword = "UNUSED Push to password"
```

Comment:
Rather than give up the visual cues helpful in the storyboard, I prefixed the identifiers with `UNUSED` to indicate that they aren't used.